astor/__init__.py

```diff
@@ -38,3 +40,38 @@
 all_symbols = symbol_data
 treewalk = tree_walk
 codegen = code_gen
+"""
+
+exec(deprecated)
+
+def deprecate():
+    def wrap(deprecated_name, target_name):
+        if '.' in target_name:
+            target_mod, target_fname = target_name.split('.')
+            target_func = getattr(globals()[target_mod], target_fname)
+        else:
+            target_func = globals()[target_name]
+        msg = "astor.%s is deprecated.  Please use astor.%s." % (deprecated_name, target_name)
+        if callable(target_func):
+            def newfunc(*args, **kwarg):
+                warnings.warn(msg, FutureWarning)
+                return target_func(*args, **kwarg)
+        else:
+            class ModProxy:
+                def __getattr__(self, name):
+                    warnings.warn(msg, FutureWarning)
```

Comment:
We need to pass proper `stacklevel` values here. I think it should 4 or 5 for module level deprecations, but I don't know the other case.